PlayerScripts/PlayerScriptsPlace/StarterPlayer/StarterPlayerScripts/CameraScript/RootCamera/ClassicCamera.rbxmx

```diff
@@ -14,6 +14,14 @@ if not VRServiceExists or not VRService then
 	VRService = UserInputService
 end
 
+local VRCameraUpdate = false
+local success, msg = pcall(function()
+	VRCameraUpdate = UserSettings():IsUserFeatureEnabled("VRCameraUpdateLua")
```

Comment:
The `IsUserFeatureEnabled` is generally not used because they have an undesirable effect if the dev copies the camera scripts and then we remove this flag.
Is this feature something that we need to be worried about turning on and off?